tir/technologies/webapp_internal.py

```diff
@@ -969,6 +1000,12 @@ def filling_branch(self, shadow_root=None, container=None):
                 if click_type > 3:
                     click_type = 1
 
+        if branch_value.strip() != self.config.branch.strip():
+            message = "Cant_fill_branch"
+            self.log.take_screenshot_log(self.driver, description=message, stack_item=self.log.get_testcase_stack())
```

Comment:
We need understand what the intentions for this change. We understood that you are tryng to test the access screens. If that's the intention, it is not TIR purpose.